src/resmom/requests.c

```diff
@@ -548,6 +548,8 @@ return_file(job *pjob, enum job_file which, int sock)
 		return (-1);
 }
 
+#define HOOK_RUNNING_IN_BACKGROUD (3)
```

Comment:
I think you meant *BACKGROUND and not *BACKGROUD.
Also, this define should be in an include file that is seen by mom_hook_func.c where inside mom_process_hooks(), instead of returning the hard-wired 3, it should be returning this macro instead.

Reply:
Done.

Reply:
as we have defined HOOK_RUNNING_IN_BACKGROUND in the header file, we can remove this #define.

Reply:
Done